com.unity.ml-agents/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to
 ### Major Changes
 
 ### Minor Changes
+Model checkpoints are now also saved as .nn files during training. (#4127)
```

Comment:
+1. I will add to this, mentioning that TrainingStatus.json will output the ML-Agents model checkpoint info.